crates/bevy_render/src/view/window/mod.rs

```diff
@@ -431,5 +405,7 @@ pub fn create_surfaces(
             };
             render_device.configure_surface(&data.surface, &data.configuration);
         }
+
+        window_surfaces.configured_windows.insert(window.entity);
```

Comment:
I thought marking windows configured would fit after configuring them instead of in the next system where they are configured only when outdated.